<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRoundedTextRectItemDialog</class>
 <widget class="QDialog" name="QtRoundedTextRectItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRoundedEditRectItem</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_font">
     <property name="text">
      <string>Font:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="button_font">
     <property name="text">
      <string>Change fo&amp;nt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_padding_left">
     <property name="text">
      <string>Padding left</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="box_padding_left"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_padding_top">
     <property name="text">
      <string>Padding top</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="box_padding_top"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_padding_right">
     <property name="text">
      <string>Padding right</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="box_padding_right"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_padding_bottom">
     <property name="text">
      <string>Padding bottom</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="box_padding_bottom"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="text"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_text_pen">
     <property name="text">
      <string>Text pen</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="button_text_pen">
     <property name="text">
      <string>Change &amp;text pen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
